### PR TITLE
Deprecate mysqlclient/MySQLdb connector support

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Relates to https://github.com/ansible-collections/community.mysql/issues/654

### CODE REVIEW

**Comment by Andersson007** [comment]
ready for review

File: README.md [hunk 1]
```diff
@@ -122,17 +122,10 @@ For MariaDB, only Long Term releases are tested.
 - pymysql 0.7.11 (Only tested with MySQL 5.7)
 - pymysql 0.9.3
 - pymysql 1.0.2 (only collection version >= 3.6.1)
-- mysqlclient 2.0.1
-- mysqlclient 2.0.3 (only collection version >= 3.5.2)
-- mysqlclient 2.1.1 (only collection version >= 3.5.2)
 
 ## External requirements
 
-The MySQL modules rely on a MySQL connector. The list of supported drivers is below:
```
Comment (dennisurtubia) [inline README.md:131]: Instead remove this lines, we couldn't warn here that it will be deprecated?

Comment (Andersson007) [reply README.md:131]: @dennisurtubia fair call, added, thanks!

**Review by markuman — APPROVED** [review]

**Comment by Andersson007** [comment]
@dennisurtubia @markuman thanks for reviewing!

**Pull request merged**